config-connector/tests/ccs-test/cmd/root.go

```diff
@@ -102,7 +108,14 @@ var (
 				return
 			}
 
-			realValues, err := finalizeValuesFromEnv(envValues, testValues)
+			// Generate the random IDs first.
+			randomId, err := util.GenerateRandomizedSuffix()
+			if err != nil {
+				log.Fatalf("error generating the randomized suffix for resource names: %v", err)
```

Comment:
Should we return if error?

Reply:
Good point. Actually log.Fatalf() will print the error and exit with code 1 so the return above is never reached. Will clean them up later.